test/DebugInfo/debug_prefix_map.swift

```diff
@@ -2,6 +2,13 @@
 // RUN: %target-swift-frontend -emit-module-path %t/Globals.swiftmodule %S/Globals.swift
 // RUN: %target-swiftc_driver -g -debug-prefix-map %/S=/var/empty -debug-prefix-map %t=/var/empty %/s -I %t -emit-ir -o - | %FileCheck %s
 
+// RUN: %empty-directory(%t) && cd %t && cp %s .
+// RUN: %target-swift-frontend -emit-module-path %t/Globals.swiftmodule %S/Globals.swift
+// RUN: %target-swiftc_driver -g -debug-prefix-map %/S=. -debug-prefix-map %t=. debug_prefix_map.swift -I %t -emit-ir -o - | %FileCheck %s --check-prefix=CHECK-RELATIVE
+
+// RUN: mkdir -p %t/nested/dir && cd %t/nested/dir && echo > foo.swift
+// RUN: %target-swiftc_driver -g -debug-prefix-map %/t/nested=. %/t/nested/dir/foo.swift -emit-ir -o - | %FileCheck %s --check-prefix=CHECK-PARTIAL-REMAP
```

Comment:
`%/t/nested/dir/foo.swift` looks like a typo

Reply:
Hrm I think it's correct, what were you expecting it to be?

Reply:
I was expecting `%t/nested/dir/foo.swift`.

Reply:
Ah I copied the pattern that folks were forcing the forward slash in paths from the other tests, only matters on windows